port_ocean/core/defaults/initialize.py

```diff
@@ -205,6 +208,12 @@ async def _initialize_defaults(
         logger.warning("No defaults found. Skipping initialization...")
         return None
 
+    if integration_config.use_provisioned_defaults:
+        logger.info("`use_provisioned_defaults` set, verifying org feature toggle")
+        org_feature_toggles = await port_client.get_organization_feature_toggles()
+        if ORG_USE_PROVISIONED_DEFAULTS_FEATURE_TOGGLE not in org_feature_toggles:
+            integration_config.use_provisioned_defaults = False
```

Comment:
wouldn't we want to log this case as well?

Reply:
Fixed!